script/Ciel/Day 4/04_01_CIEL04_1B.txt

```diff
@@ -390,7 +391,8 @@ Please take care on your way to school today."
 [sha:61e7ddc7a7c36ec8042a71855874e5d64e739e1a]{
 -- Page 68, Offset 8334.
 -- 翡翠は深々と礼をして俺を送り出してくれた。
-Bowing especially deeply, she sends me on my way.
+I thank Senpai from the bottom of my heart for that smile of hers,
+ and begin to make my way back to the Tohno mansion.
```

Comment:
Uh what's goin on here? this is a totally different line

Reply:
Not sure what happened here, I'll just revert it